--- a/knewsticker/CMakeLists.txt
+++ b/knewsticker/CMakeLists.txt
@@ -13,7 +13,7 @@
 kde4_add_ui_files(knewsticker_SRCS visualsettings.ui feedsettings.ui)
 kde4_add_plugin(plasma_applet_knewsticker ${knewsticker_SRCS})
 
-target_link_libraries(plasma_applet_knewsticker ${PLASMA_LIBS} ${KDE4_SYNDICATION_LIBS})
+target_link_libraries(plasma_applet_knewsticker ${PLASMA_LIBS} ${KDE4_SYNDICATION_LIBS}  ${KDE4_KDECORE_LIBS} ${QT_QTCORE_LIBRARY} ${KDE4_KDEUI_LIBS})
 
 install(TARGETS plasma_applet_knewsticker DESTINATION ${PLUGIN_INSTALL_DIR})
 
